prerequisite_changes.py

```diff
@@ -228,7 +232,53 @@ def main():
         replace("TAF/testScenarios/functionalTest/V2-API/core-command/device/GET-Positive.robot",
                 "== 6",
                 "== ${length}")
+    if service == "4_ping_response_time" or service=="performance-metrics-collection":
```

Comment:
Is this the name of ping service in TAF? @bhavikkothari-eaton

Reply:
It is the **ping_response_time.robot** test which measure the ping response time of ping API for each edgex service.
service list 
1. core-data
2. core-metadata
3. core-command
4. support-scheduler
5. support-notification
6. sys--mgmt-agent
7. device-rest
8. device-rest
9. app-service
10. device-virtual